src/shapes/text.class.js

```diff
@@ -927,9 +931,9 @@
       var box = {
         width: width,
         left: 0,
-        height: style.fontSize,
+        height: style.fontSize + Math.abs(style.deltaY),
```

Comment:
the height of the box is the fontSize. We should not move from this assumptions.